opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -210,15 +217,12 @@ message ScopeProfiles {
 // for ease of understanding data migration, it is not intended that pprof:Profile and
 // OpenTelemetry:Profile encoding be wire compatible.
 message Profile {
-  // A description of the samples associated with each Sample.value.
-  // For a cpu profile this might be:
-  //   [["cpu","nanoseconds"]] or [["wall","seconds"]] or [["syscall","count"]]
+  // The type and unit of all Sample.values in this profile.
+  // For a cpu or off-cpu profile this might be:
+  //   ["cpu","nanoseconds"] or ["off_cpu","nanoseconds"]
   // For a heap profile, this might be:
```

Comment:
```suggestion
  // See ScopeProfiles.default_sample_type for some possible values.
  // Optional, ScopeProfiles.default_sample_type is used when missing.
```

Reply:
Alternatively, have `ScopeProfiles.default_sample_type` refer to `Profile.sample_type` for possible values, we shouldn't duplicate the values in two places.

Reply:
Applied with https://github.com/open-telemetry/opentelemetry-proto/pull/649/commits/5f0767dd04a875015b994ef9ac11800adf9a357d